app/Models/User.php

```diff
@@ -556,4 +558,24 @@ public function scopeVerified(Builder $query): Builder
     {
         return $query->where('Permissions', '>', 0);
     }
+
```

Comment:
These new functions all belong in _ActsAsCommunityMember.php_.

Use `relatedUsers()`, `followedUsers()`, etc as a reference to build the new relation.

Reply:
Hah, this makes it so much simpler.

Reply:
resolved with:
```
/**
     * Check if two users are mutually following each other.
     */
    public function isFriendsWith(User $user): bool
    {
        return $this->isFollowing($user) && $user->isFollowing($this);
    }
```